app.js

```diff
@@ -262,6 +354,32 @@ app.get('/hubs/:hubSlug/activities', function(req, res){
   }
 })
 
+app.get('/hubs/:hubSlug/devices', function(req, res){
+  hubSlug = req.params.hubSlug
+  harmonyHubClient = harmonyHubClients[hubSlug]
+
+  if (harmonyHubClient) {
+    var devices = [];
+    res.json({devices: cachedHarmonyDevices(hubSlug).map(function(device) {
```

Comment:
Map the slugs to an array on it's own line, and then use this array in the response JSON. Just to clean it up a little.